### PR TITLE
[ Tests ] Add new test for ProcessHandlerFilter

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Add new test.

### Modifications

- Add new test for ProcessHandlerFilter

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.web;
+
+import static org.mockito.Mockito.*;
```
Comment (codelipenghui) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java:21]: Please avoid star import

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        processHandlerFilter2.doFilter(mockHttpServletRequest2, mockHttpServletResponse, spyFilterChain);
+        verify(spyFilterChain).doFilter(mockHttpServletRequest2, mockHttpServletResponse);
+        clearInvocations(spyFilterChain);
+        // request has APPLICATION_OCTET_STREAM content-type
+        doReturn(Sets.newHashSet("Interceptor1","Interceptor2")).when(mockConfig).getBrokerInterceptors();
+        HttpServletRequest mockHttpServletRequest3 = mock(HttpServletRequest.class);
+        doReturn(MediaType.APPLICATION_OCTET_STREAM).when(mockHttpServletRequest3).getContentType();
+        ProcessHandlerFilter processHandlerFilter3 = new ProcessHandlerFilter(mockPulsarService);
+        processHandlerFilter3.doFilter(mockHttpServletRequest3, mockHttpServletResponse, spyFilterChain);
+        verify(spyFilterChain).doFilter(mockHttpServletRequest3, mockHttpServletResponse);
```
Comment (codelipenghui) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java:82]: Could you please refine these lines? Looks only the MediaType different here.

**Review by codelipenghui — CHANGES_REQUESTED** [review]

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java (hunk 2 above)
Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java:82]: OK, I will refine these.

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java (hunk 1 above)
Comment (mattisonchao) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java:21]: Ok, I will fix it.

**Comment by mattisonchao** [comment]
@codelipenghui  PTAL

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**